### PR TITLE
[Release] Add release notes

### PR DESCRIPTION
**Pull request by benoitgrelard**
Todo:
- [x] Check final versions
- [x] Update `NavigationMenu` PR id

### CODE REVIEW

File: data/primitives/overview/releases.mdx [hunk 1]
```diff
@@ -7,6 +7,72 @@ metaDescription: Radix Primitives releases and their changelogs.
...
+
+<PackageRelease name="Context Menu" version="0.1.5" />
+
+- Prevent `DropdownMenu.TriggerItem` click from firing twice <PRLink id={1057} />
+- Improve idle performance <PRLink id={1040} />
+
+<PackageRelease name="Dialog" version="0.1.6" />
+
+- Improve pointer-events management <PRLink id={1079} />
+- Fix labelling issues when no `Description` is used <PRLink id={1098} />
```
Comment (jjenzz) [inline data/primitives/overview/releases.mdx:39]: Do we want to highlight the breaking changes here? We said we would be more explicit on those in the release notes in future.

- `Title` is now a required part so will throw an error if not used.
- `aria-describedby={undefined}` must be passed if no description is needed.

File: data/primitives/overview/releases.mdx [hunk 2]
```diff
@@ -7,6 +7,72 @@ metaDescription: Radix Primitives releases and their changelogs.
...
+
+<PackageRelease name="Context Menu" version="0.1.5" />
+
+- Prevent `DropdownMenu.TriggerItem` click from firing twice <PRLink id={1057} />
+- Improve idle performance <PRLink id={1040} />
+
+<PackageRelease name="Dialog" version="0.1.6" major />
+
+- Improve pointer-events management <PRLink id={1079} />
+- [**Breaking**] `Dialog.Title` is now a required part so will throw an error if not used. `aria-describedby={undefined}` must be passed to `Dialog.Content` if no description is needed. <PRLink id={1098} />
```
Comment (jjenzz) [inline data/primitives/overview/releases.mdx:39]: This has made me wonder if we should throw that error in dev mode only... currently doing it always.... 🤔

Comment (benoitgrelard) [reply data/primitives/overview/releases.mdx:39]: We don't right?

image

Comment (jjenzz) [reply data/primitives/overview/releases.mdx:39]: ooooh niiiice the condition is around the component render 👍 I checked inside the comps! haha.

**Review by jjenzz — APPROVED** [review]

**Pull request merged**